packages/zoe/src/zoeService/types.js

```diff
@@ -271,7 +271,7 @@
  */
 
 /**
- * @typedef {Partial<ProposalRecord>} Proposal
+ * @typedef {Partial<ProposalRecord>}   Proposal
```

Comment:
bad whitespace

Reply:
not bad, just misunderstood

Reply:
Seeing this, I immediately lept to "I'm not bad, I'm just drawn that way"

![image](https://user-images.githubusercontent.com/273868/148462950-7a0ee68a-10c6-4bef-a32d-cd18c301fd85.png)

Done